docs/pages/database-access/rbac.mdx

```diff
@@ -1,22 +1,39 @@
 ---
-title: Database Access RBAC
+title: Database Access Controls
 description: Role-based access control (RBAC) for Teleport database access.
 ---
 
-Role-based access control (or RBAC, for short) allows administrators to set up
-granular access policies for databases connected to Teleport.
+**Database Access Controls** is a Teleport feature that lets you configure
+role-based access controls for databases and the data within them. With Database
+Access Controls, you can ensure that users only have permissions to manage the
+data they need.
 
-An example of a policy could be, *"database administrators have access to
-everything, QA team and engineers have full access to staging databases, and
-engineers can gain temporary access to the production database in case of
-emergency"*.
+Access Controls encompasses two levels of granularity:
 
-For a more general description of Teleport roles and examples see [RBAC](../access-controls/introduction.mdx), as
-this section focuses on configuring RBAC for database access.
+- **Databases:** databases enrolled with your Teleport cluster.
```

Comment:
Nit: Maybe clarify this as "database servers"? Cause logical databases within DBMS are also called "databases".